Makefile

```diff
@@ -325,9 +326,10 @@ dist/js/subtitles-octopus-worker.js: src/subtitles-octopus-worker.bc src/pre-wor
 		-s WASM=1 \
 		$(EMCC_COMMON_ARGS)
 
-dist/js/subtitles-octopus-worker-legacy.js: src/subtitles-octopus-worker.bc src/pre-worker.js src/SubOctpInterface.js src/post-worker.js build/lib/brotli/js/decode.js
+dist/js/subtitles-octopus-worker-legacy.js: src/subtitles-octopus-worker.bc src/pre-worker.js src/SubOctpInterface.js src/post-worker.js build/lib/brotli/js/decode.js build/lib/brotli/js/polyfill.js
 	mkdir -p dist/js
 	emcc src/subtitles-octopus-worker.bc $(OCTP_DEPS) \
+		--pre-js build/lib/brotli/js/polyfill.js \
 		--pre-js src/pre-worker.js \
 		--pre-js build/lib/brotli/js/decode.js \
```

Comment:
I guess with our polyfills being merged into `src/pre-worker.js` we need to add `build/lib/brotli/js/polyfill.js` first so we can use the polyfills from brotli — but this means brotli's polyfill versions take precendence over ours. Can you prepend a commit which moves our polyfills into a separate file as you propsed (being only used by the legacy worker, if that's a safe assumption)? Then we can add brotli polyfills after ours.

Reply:
Only these? https://github.com/libass/JavascriptSubtitlesOctopus/blob/f045d1cf7bcda011d3306b4dd350b22c80a0c4f8/src/pre-worker.js#L1-L50

Or also this? https://github.com/libass/JavascriptSubtitlesOctopus/blob/f045d1cf7bcda011d3306b4dd350b22c80a0c4f8/src/pre-worker.js#L188-L203

Reply:
Both.